Added Snowflake.Data.Client.SnowflakeDbDataAdapter()

mkellsy opened the pull request:
I am from Encompass. We had discussed data adapters in our conversations. I have wrote a read only data adapter that can be used to create System.Data.DataTables.

howryu commented:
@mkellsy Thanks for your contribution. Looks good to me. I will add some tests to it and merge the change afterwards. 

Please note that I have to wrap up your change and submit another pr to from snowflakedb repo since tests cannot be run under a branch outside of snowflakedb repo.

mkellsy commented:
I added the changes per our phone call. I removed the "not supported" exceptions from the Insert, Delete and Update commands. I also added the copyright header to the top of the file.

howryu commented:
I merge this to master. Target for 1.0.3
